packages/react/src/lib/accessibility/FocusZone/AutoFocusZone.tsx

```diff
@@ -19,19 +18,16 @@ export default class AutoFocusZone extends React.Component<AutoFocusZoneProps> {
     firstFocusableSelector: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
   }
 
-  static handledProps = _.keys(AutoFocusZone.propTypes)
+  static handledProps = _.keys(AutoFocusZone.propTypes) as any
```

Comment:
💣 

`_.keys()` and `Object.keys()` returns `string[]` in TS:

![image](https://user-images.githubusercontent.com/14183168/66136253-72f93f00-e5fb-11e9-96f5-ba95c3123f8e.png)

So type cast is required is there as typings for `getUnhandledProps()` are more strict.